Doc/library/stdtypes.rst

```diff
@@ -2841,6 +2841,13 @@ objects.
          optional *sep* and *bytes_per_sep* parameters to insert separators
          between bytes in the hex output.
 
+   .. method:: resize(size)
+
+      Resize the :class:`bytearray` to contain *size* bytes with a NULL byte
+      following.
```

Comment:
Please describe the behavior when the buffer becomes larger (fill with null bytes), and when the buffer becomes shorter (truncate).

Reply:
The resize doesn't currently do any filling of the buffer in Python / leaves it up to the allocation functions, it just does a malloc + copy or realloc followed by setting the last byte to null: https://github.com/python/cpython/blob/main/Objects/bytearrayobject.c#L226-L249